robot_frame: "base_link"
max_velocity: 0.8
min_velocity: 0.0
max_yawrate: 0.6
max_acceleration: 100.0
max_d_yawrate: 100.0
